extensions/wikia/Editcount/SpecialEditcount_body.php

```diff
@@ -186,6 +189,7 @@ function editsArchived( $uid ) {
 
 			$wgMemc->set( $key, $arcount, self::CACHE_TIME );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)